### PR TITLE
S 68294 commit stream style change side bar

### PR DESCRIPTION
**Pull request by mirvin**

### CODE REVIEW

File: src/app/client/commitsContainer.html [hunk 1]
```diff
@@ -3,8 +3,11 @@
   display: none;
 }
 </style>
-
-<link type="text/css" rel="stylesheet" href="{{resourcePath}}css/commitStream.css">
+<!-- Since this file is for cs classic, we will still point to light-->
+<!-- doesn't this force us to get styles from  cs instance still for classic?
+This brings up another problem.  The
+-->
+<link type="text/css" rel="stylesheet" href="{{resourcePath}}css/commitStream-Light.css">
```
Comment (jallers) [inline src/app/client/commitsContainer.html]: What happens with this stylesheet?

File: src/app/client/commitsContainer-Light.html [hunk 2]
```diff
@@ -0,0 +1,43 @@
+<style>
+.commitStream-panel {
+  display: none;
+}
+</style>
+
+<link type="text/css" rel="stylesheet" href="css/commitStream-Light.css">
```
Comment (jallers) [inline src/app/client/commitsContainer-Light.html]: Does this reference VersionOne's hosted css? We probably need to figure out if this works for shared static.

File: src/app/client/commitsContainer.html (hunk 1 above)
Comment (mirvin) [reply src/app/client/commitsContainer.html]: resource path is on the Commitstream server.  It then gets loaded and send down to core in the case of the the OG commistream users as apposed to our newer way of getting styles from core.

File: src/app/client/commitsContainer-Light.html (hunk 2 above)
Comment (mirvin) [reply src/app/client/commitsContainer-Light.html]: yes

**Pull request merged**